### DISCUSSION TITLE
Mehr als 12 Geräte zur Strommessung mit SAE im Sunny Portal möglich?

### DISCUSSION THREAD

**Discussion by BurtGummer0815** [camueller/SmartApplianceEnabler#498] [Q&A]
Guten Abend in die Runde.

Arbeite mich grade etwas ein mit SAE.
Im Sunny Portal bei den Verbrauchern steht die Bemerkung, dass 24 Geräte hinzugefügt werden können, genau:

"Von den 24 Geräten können maximal 12 Geräte vom Sunny Home Manager aktiv gesteuert werden. Über die Grenze von 12 Geräten hinaus, können weitere Geräte bis zur maximalen Anzahl von 24 Geräten überwacht und visualisiert, aber nicht geschaltet werden."

Habe mehrere Gosund Steckdosen, die ich gerne einpflegen würde. Allerdings ist bei 12 Geräten Schluss.
Kann SAE nicht mehr?
Die Steckdosen sollen nicht durch Sunny Portal geschaltet werden und dienen lediglich der Messung.

Oder mache ich etwas falsch bei den Gerätetypen?

**Comment by camueller** [comment]
Da wäre ein Blick in die SAE-Doku empfehlenswert: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SunnyPortal_DE.md#maximale-anzahl-der-ger%C3%A4te-im-sunny-portal

**Reply by BurtGummer0815** [reply]
Oh, OK, das war mir gar nicht aufgefallen. Der "Trick" funktioniert aktuell noch?
Reicht es lediglich das Gerät mit Seriennummer im SAE anzulegen oder muss auch der Zähler selber mit IP und JSON eingetragen werden?

**Reply by camueller** [reply]
Schalter under Zähler sind nicht erforderlich, sind ja SAE-Interna.

**Reply by BurtGummer0815** [reply]
Scheint wohl leider nicht mehr zu funktionieren? 
Oder mache ich noch etwas falsch?
Habe sogar 2 Geräte gelöscht, es erscheinen vorne nicht die Kästchen zum markieren. Füge ich dann ein Gerät über das Plus Zeichen hinzu, kann ich keine weiteren mehr hinzufügen, weil aus dem Plus ein rotes Kreuz wird :(

![Sunny](https://user-images.githubusercontent.com/127900848/225421098-49c24eae-331b-476d-9847-6dc6ed305f6e.jpg)

**Reply by BurtGummer0815** [reply]
Habe mittlerweile alle Geräte im Portal gelöscht...
Keine Chance, es kommt kein Auswahlfenster zum anhaken :(
Frage mich nur, warum da noch der Button mit "hinzufügen" steht....

**Reply by BurtGummer0815** [reply]
Habe jetzt mal SMA geschrieben, obwohl ich da wenig Hoffnung habe. Support ist da eine Katastrophe.
Am Telefon wollen die nicht helfen, die info@sma will auch nicht, dafür muss man sich beim Support noch Mal extra registrieren.
Das geht natürlich auch nicht automatisch, sondern wird von Hand von einem Mitarbeiter freigeschaltet.
So langsam wundert mich bei SMA nichts mehr....

Die Regierung möchte einem am Liebsten per Smartmeter zu jeder Zeit den Stromabdrehen und bei SMA platzt der Pentium 1 Server bei mehr als 12 Smart Devices aus allen Nähten....

@camueller 
Denke du kannst deine Doku mal abändern, dass es mit dem Hinzufügen von mehr als 12 Geräten nicht mehr funktioniert.
Sollte ich bei SMA was erreichen (Hoffnung gleich Null, auch aus früheren Anfragen) melde ich mich hier noch einmal.

**Reply by camueller** [reply]
Habe bei SMA nachgefragt. Tatsächlich haben sie dieses "Schlupfloch" geschlossen. Ich werde die Doku mal anpassen.

**Reply by BurtGummer0815** [reply]
Ok, danke für die Info.
Hatte nun alle Devices gelöscht, wieder hinzugefügt und gelöscht über die Eigenschaften. Das dauert ja pro Gerät 5 Minuten (kein Scherz).
Toll, jetzt hatte ich mir extra noch mal Shellys bestellt.

Es gibt keine Chance, SAE so abzuändern, dass Steckdosen mit Tasmota lediglich als Meter und nicht als Schalter erkannt werden über den Homemanager?
Kaufe ich mir Steckdosen von SMA und ich glaube auch die AVM Steckdosen werden unterstützt, kann ich 24 (22) Devices hinzufügen?

**Reply by camueller** [reply]
Am SAE kann man nichts machen in dieser Hinsicht.
